counsel.el

```diff
@@ -731,6 +731,25 @@ With a prefix arg, restrict list to variables defined using
       (when doc
         (lv-delete-window)))))
 
+;;** `counsel-tab-bar'
```

Comment:
This section is under `;;* Elisp symbols`, where it does not belong. Please move it under `;; Misc. Emacs` or similar.